.github/workflows/thank-contributor.yml

```diff
@@ -12,14 +9,19 @@ jobs:
       github.event.pull_request.user.login != 'dependabot[bot]'
     runs-on: ubuntu-latest
     steps:
+      - name: Sleep for 120 seconds
+        run: sleep 120s
+        shell: bash
       - name: Thank contributor for contribution
         env:
           ISSUES_BOT_TOKEN: ${{ secrets.ISSUES_BOT_TOKEN }}
         run: |
           username="${{ github.event.pull_request.user.login }}"
           repo="${{ github.repository }}"
-          response=$(curl -s -o /dev/null -w "%{http_code}" -H "Authorization: Bearer $ISSUES_BOT_TOKEN" "https://api.github.com/repos/$repo/collaborators/$username")
-
+          if [[ "$username" =~ [[:punct:]] ]]; then
+            message="Hi @$username. :wave:\n\nYour PR has been approved and merged. :tada:\n\nThank you for your contribution to this project. :heart:\n\nKeep up the great work!"
```

Comment:
For both this section response and the one on line 26, should still still be "project" and not "starter.dev"?